implementations/rust/ockam/ockam_node/src/node.rs

```diff
@@ -31,16 +32,57 @@ pub fn start_node() -> (Context, Executor) {
 }
 
 /// Utility to setup tracing-subscriber from the environment
+#[cfg(not(feature = "console"))]
+fn setup_tracing() {
+    #[cfg(feature = "std")]
+    {
+        let filter = Box::new(EnvFilter::try_from_env("OCKAM_LOG").unwrap_or_else(|_| {
+            EnvFilter::default()
+                .add_directive(LevelFilter::INFO.into())
+                .add_directive("ockam_node=info".parse().unwrap())
+        }));
+
+        if tracing_subscriber::registry()
+            .with(filter)
+            .try_init()
+            .is_err()
+        {
+            debug!("Failed to initialise tracing_subscriber.  Is an instance already running?");
+        }
+    }
+}
+
+#[cfg(feature = "console")]
+/// Utility to setup tracing-subscriber from the environment.
+/// Enables usage with the `tokio-console` diagnostic toolkit.
+///
+/// # Usage
+/// The `tokio_unstable` cfg flag, which enables experimental APIs in Tokio, must
+/// be enabled. It can be enabled by setting the `RUSTFLAGS` environment variable
+/// at build-time:
+/// ```shell
+/// $ RUSTFLAGS="--cfg tokio_unstable" cargo build
```

Comment:
I'm not sure what this is referring to, since we don't have a `tokio_unstable` feature?

EDIT: Ah, This is cfg not features. Hmm...